cmd/lotus-shed/indexes.go

```diff
@@ -682,68 +736,34 @@ var backfillMsgIndexCmd = &cli.Command{
 
 var pruneMsgIndexCmd = &cli.Command{
 	Name:  "prune-msgindex",
-	Usage: "Prune the msgindex.db for messages included before a given epoch",
+	Usage: "Prune the msgindex.db for messages older than a given height",
 	Flags: []cli.Flag{
 		&cli.IntFlag{
-			Name:  "from",
-			Usage: "height to start the prune; if negative it indicates epochs from current head",
+			Name:  "older-than",
+			Value: 0,
+			Usage: "height to start pruning events older than this epoch, 0 means start from head",
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		api, closer, err := lcli.GetFullNodeAPI(cctx)
+		srvc, err := lcli.GetFullNodeServices(cctx)
 		if err != nil {
 			return err
 		}
 
-		defer closer()
-		ctx := lcli.ReqContext(cctx)
+		api := srvc.FullNodeAPI()
+		closer := srvc.Close
```

Comment:
maybe just keep the pattern the same across these, you have some churn in naming and you have an extra `closer` here; the one below in `pruneTxHashCmd` is clean:

```go
		srv, err := lcli.GetFullNodeServices(cctx)
		if err != nil {
			return err
		}
		defer srv.Close() //nolint:errcheck

		api := srv.FullNodeAPI()
		ctx := lcli.ReqContext(cctx)
```

Reply:
Done.